exp/erc721events/transfer_v2.go

```diff
@@ -0,0 +1,24 @@
+package erc721events
+
+import (
+	"math/big"
+	"github.com/indexsupply/x/abi"
+)
+
+type TransferV2 struct {
```

Comment:
mentioned in chat, but i'll post here too: I like this direction the best. Ideally we can not expose Item. Perhaps we can make it private. Getting a [20]byte from `From` and `To` seems ideal.